samples/mp4-decode/mp4_demuxer.js

```diff
@@ -111,7 +111,7 @@ class Writer {
   }
 }
 
-export class MP4Demuxer {
```

Comment:
why is this no longer needed?

Reply:
There's probably a way to keep this file as a module, and to still use MP4Box.js as a script. I had tried Dan's suggestion of making the worker a module, but I still ran into issues. Making everything a script was the simplest way that I could get the demo to work.